src/actions.js

```diff
@@ -36,6 +36,18 @@ export function fetchCalculationParamsList(className, instanceClassName, instanc
 
 export function fetchCalculationRules(params) {
     let filter = !!params !== [] ? params : null;
+    if (filter === undefined) {
+        filter = []
+    }
+    
+    var pathname = window.location.pathname 
+    if (pathname.includes('contributionPlanBundles')) {
```

Comment:
Why didn't you use <https://github.com/openimis/openimis-be-calcrule-capitation_payment_py/blob/d86a07b349ee36fe0cbd129d2da60e696831d93d/calcrule_capitation_payment/calculation_rule.py#L30 , my apology for the not clear requirement

Reply:
account payable for the payment_plan
account_receivable for the contribution plan
https://openimis.atlassian.net/wiki/spaces/OP/pages/3046277126/Calculation+rules#type

Reply:
ok, then this could be better solution. I have to fix this. @delcroip

Reply:
fixed!